src/utilities/findBreakingChanges.js

```diff
@@ -105,6 +110,80 @@ export function findTypesThatChangedKind(
   return breakingChanges;
 }
 
+/**
+ * Given two schemas, returns an Array containing descriptions of any breaking
+ * changes in the newSchema related to removal of an arg.
```

Comment:
Seems that "removal of an arg" is only a partial description of what this does. I'd suggest something like:

```
/**
 * Given two schemas, returns an Array containing descriptions of any breaking
 * changes in the newSchema related to arguments (such as removal or change
 * of type of an argument, or a change in an argument's default value).
 */
```